client/src/components/TopSongGallary.jsx

```diff
@@ -0,0 +1,52 @@
+import { Box, Image, ImageBox, ImageBoxContainer, SongP, Songartist, Title, TopASongCountainer, TopHSongCountainer, } from './TopSongGallaryStyle';
+import albumcover from '../assets/eden.jpg'
+import cover2 from '../assets/aden.jpg'
+import cover3 from '../assets/nasif.jpg'
+import cover4 from '../assets/amrdiab.jpg'
+
+export default function TopSongGallary() {
+  return (
```

Comment:
please orgenize the code with tabs, so it will be easy to see what is nested inside what

Reply:
organized